site/content/user-guide/mqtt-adapter.md

```diff
@@ -194,6 +198,13 @@ For example, if the [HonoExampleApplication]({{< relref "dev-guide/java_client_c
        "brightness" : 79
     }
 
+If the command is a *one-way* command, it will arrive as follows:
+
+    $ c///q//setBrightness {
```

Comment:
are your sure about the `c/`? IMHO the MQTT adapter always publishes to `control/`

Reply:
Actually, IMHO it depends on to which topic the client originally subscribed.
It was like that in the already existing example, but it does not match with all the other topics on that page (which I didn't check).

So, good point - I will change this to `control` instead of the (also valid) `c`. This should eliminate the confusion.